README.md

```diff
@@ -128,6 +148,12 @@ which has been used by many to that purpose.
 
 In progress, pending on https://github.com/coq/coq/pull/16261
 
+### Workspace Integration
+
+Don't worry about ever having to build your project again, `coq-lsp` will
```

Comment:
Is this really true?

Reply:
That's under "planned features"; indeed that section is quite confusing.

But yes, that will happen soon.